pywerview/cli/helpers.py

```diff
@@ -24,305 +24,304 @@
 from pywerview.functions.hunting import UserHunter, ProcessHunter, EventHunter
 
 def get_adobject(domain_controller, domain, user, password=str(),
-                lmhash=str(), nthash=str(), queried_domain=str(), queried_sid=str(),
-                queried_name=str(), queried_sam_account_name=str(), ads_path=str(),
-                custom_filter=str()):
+				lmhash=str(), nthash=str(), queried_domain=str(), queried_sid=str(),
```

Comment:
Tab and space mixing is not a good cocktail, I think you should use space to be more consistent with the rest of the project

Reply:
check https://github.com/the-useless-one/pywerview/pull/34/commits/0e18889e0961baaa16ca9e5532f9ac472cc73777

Reply:
Nice, but I've just saw that the original file is also a mix between space and tab haha get your sh*t together @the-useless-one

Reply:
- Python2 allows mix between tab and space in a same file
- Python3 want consistency in the same file

The original file was fine in python2 but not python3. Now all the project should be "space" formated so python3 OK